utils/src/main/scala/za/co/absa/enceladus/utils/modules/SourceId.scala

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2018 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.enceladus.utils.modules
```

Comment:
It was moved because it will probably be used by more modules further on. Now, it's not only about the plugin, since it is used is more ways

Reply:
@benedeki @dk1844 It seems like SourceId may not be the best name to represent either Standardization or Conformance. The issue from this is that on the combined job it would return from either of the two.

Reply:
Renaming would probably be more explanatory. I agree.
```JobPart```
```JobPhase```
...

Regarding why it was moved - now it's used in (buil-in) plugin and also in SparkJobs itself. I found it usefull to have it defined once. But if agreement would arise, that the unification is over-engineering, no issue to revert. 😉

Reply:
I had a proposal for `SourceAppId` or `SourceAppName`. I know that when combining the two it becomes a bit weird but still, only one part is running and it should be viewed as a combination of two and not as a standalone Spark Job. It is more of a convenience that they will run everything from one command.